packages/braid-design-system/src/lib/components/Toggle/Toggle.tsx

```diff
@@ -60,137 +83,151 @@ export const Toggle = forwardRef<HTMLInputElement, ToggleProps>(
+                  lightness.lightMode === 'dark',
+              }}
+            />
+            <FieldOverlay className={styles.icon}>
+              <IconTick tone="formAccent" size="fill" />
+            </FieldOverlay>
+            <FieldOverlay
+              variant="focus"
+              borderRadius="full"
+              className={styles.focusOverlay}
+            />
+            <FieldOverlay
+              variant="formAccent"
+              borderRadius="full"
+              className={!on ? styles.hoverOverlay : undefined}
+            />
           </Box>
         </Box>
-        <Box
-          component="label"
-          htmlFor={id}
-          // Todo - Replace paddings with flex-gap after browser policy change
-          /*
-          Apply padding by default to prevent padding disappearing
-          during partial completion of togglePosition and align props in Playroom
-          */
-          paddingLeft={
-            appliedTogglePosition === 'trailing' ? undefined : 'xsmall'
-          }
-          paddingRight={
-            appliedTogglePosition === 'leading' ? undefined : 'xsmall'
+      </Box>
+    );
+
+    const ToggleLabel = () => (
+      <Box
+        component="label"
+        htmlFor={id}
+        // Todo - Replace paddings with flex-gap after browser policy change
+        /*
+        Apply padding by default to prevent padding disappearing
+        during partial completion of togglePosition and align props in Playroom
+        */
+        paddingLeft={
+          appliedTogglePosition === 'trailing' ? undefined : 'xsmall'
+        }
+        paddingRight={
+          appliedTogglePosition === 'leading' ? undefined : 'xsmall'
+        }
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        flexGrow={align === 'justify' ? 1 : undefined}
+        userSelect="none"
+        cursor="pointer"
+        className={virtualTouchable}
+      >
+        <Text
+          baseline={bleedY}
+          weight={on ? 'strong' : undefined}
+          size={size}
+          align={
+            align === 'justify' && appliedTogglePosition === 'leading'
+              ? 'right'
+              : undefined
           }
-          display="flex"
-          flexDirection="column"
-          justifyContent="center"
-          flexGrow={align === 'justify' ? 1 : undefined}
-          userSelect="none"
-          cursor="pointer"
-          className={[styles.label[size], virtualTouchable]}
         >
-          <Text
-            baseline={false}
-            weight={on ? 'strong' : undefined}
-            size={size}
-            align={
-              align === 'justify' && appliedTogglePosition === 'leading'
-                ? 'right'
-                : undefined
-            }
-          >
-            {label}
-          </Text>
-        </Box>
+          {label}
+        </Text>
+      </Box>
+    );
+
+    return (
+      <Box
+        position="relative"
+        zIndex={0}
```

Comment:
The `zIndex` and `position="relative"` attributes are preventing touch targets being clipped by neighbouring selectable elements on the page. The toggle should be brought to the front on hover with the `styles.root` class